R/UK.R

```diff
@@ -169,7 +164,7 @@ UK <- R6::R6Class("UK", # rename to country name
           level_2_region_code = .data$areaCode
         ) %>%
         # Join local authority codes to level 1 regions
-        left_join(self$authority_lookup_table,
+        left_join(covidregionaldata::uk_authority_lookup_table,
```

Comment:
this doesn't need to have such a different structure to everything else does it?